github/ci/prow-deploy/kustom/overlays/kubevirt-prow-control-plane/patches/JsonRFC6902/crier_deployment.yaml

```diff
@@ -14,9 +14,15 @@
 - op: remove
   path: /spec/template/spec/containers/0/env/1
 
-# removes volume and moint for aws-iam-token
+# removes volume and mount for aws-iam-token
 - op: remove
-  path: /spec/template/spec/containers/0/volumeMounts/9
+  path: /spec/template/spec/containers/0/volumeMounts/10
+- op: remove
+  path: /spec/template/spec/volumes/10
+
+  # removes kubeconfig-k8s-infra-kops-prow-build mount and volume
+- op: remove
+  path: /spec/template/spec/containers/0/volumeMounts/5
```

Comment:
IIRC the indices of the modified items are not zero based, but one based, so you might also need to take that into account. Should be visible when you count the list items manually.

Just to add another annoyance here :-(

Reply:
Forget that, I think my memory was failing me.